client/webserver/api.go

```diff
@@ -853,9 +853,11 @@ func (s *WebServer) writeAPIError(w http.ResponseWriter, err error) {
 	if errors.As(err, &cErr) {
 		code = cErr.Code()
 	}
+
+	rawErr := core.Unwrap(err)
```

Comment:
In this context, you can see why `Unwrap` is not a great name for a `core` package function.  If there were an errors package it could work.